CMakePresets.json

```diff
@@ -109,6 +109,22 @@
       },
       "hidden": true
     },
+    {
+      "name": "Scarlett",
```

Comment:
When I merge this change to the xbox branch, I'll need to rename the existing "XboxOne" and "Scarlett" to something else since those are PC builds with tooling support.

Reply:
Fixing this in [321](https://github.com/microsoft/DirectXTex/pull/321).